server/src/main/java/org/opensearch/action/bulk/TransportBulkAction.java

```diff
@@ -631,7 +630,7 @@ protected void doRun() {
                 final boolean isOnlySystem = isOnlySystem(bulkRequest, clusterService.state().metadata().getIndicesLookup(), systemIndices);
```

Comment:
Can we delay this `isOnlySystem` call as well? Like the `ramBytesUsed()` it's not needed for anything other than the releasable.

Reply:
Keeping this consistent with the other interface for Node level accounting ``markCoordinatingOperationStarted(LongSupplier bytes, boolean forceExecution)`` as it is supplied with a precomputed boolean, being used elsewhere too. 

Also given ``isOnlySystem`` streams over a precomputed collection (unmodifiable set) of different indices being updated in that bulk request, we don’t expect this number (indices in a single request) to grow too high to be an overhead.